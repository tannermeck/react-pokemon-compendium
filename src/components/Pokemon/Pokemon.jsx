function Pokemon(){
    return (
        <h1>Hello from Pokemon</h1>
    )
}
export default Pokemon;