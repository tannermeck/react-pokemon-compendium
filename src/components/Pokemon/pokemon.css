ul {
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
}
.pokeContainer {
    border: 2px solid black;
    box-shadow: 2px 2px 2px 1px black;
    width: fit-content;
    margin: 20px;
}
.pokemonName {
    color: white;
    list-style: none;
    text-transform:capitalize;
    font-weight: bolder;
    text-align: center;
}
.pokemonImage {
    height: 200px !important;
    width: 200px !important;
}