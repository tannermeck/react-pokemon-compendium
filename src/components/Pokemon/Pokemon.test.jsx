import { render } from '@testing-library/react';
import Pokemon from './Pokemon.jsx';

it('should render a pokemon component', () => {
    const pokemon = [{_id: "5ff4fb7cd89993a89cc6544f", pokemon: 'butterfree', url_image: 'www.pokemon.com', type_1: 'bug', type_2: 'flying', ability_1: 'compound-eyes', ability_2: 'NA', defense: 50 }]
    const { container } = render(<Pokemon pokemon={pokemon} />)
    expect(container).toMatchSnapshot();
})