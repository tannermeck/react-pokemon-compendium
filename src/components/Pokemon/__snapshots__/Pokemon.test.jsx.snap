// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a pokemon component 1`] = `
<div>
  <ul>
    <div
      class="pokeContainer"
    >
      <li
        class="pokemonName"
      >
        butterfree
      </li>
      <img
        alt="butterfree"
        class="pokemonImage"
        src="www.pokemon.com"
      />
      <p
        class="pokemonType"
      >
        Type: bug/flying
      </p>
      <p
        class="pokemonType"
      >
        Ability: compound-eyes
      </p>
      <p
        class="pokemonType"
      >
        Defense Level: 
        50
      </p>
    </div>
  </ul>
</div>
`;
